### PR TITLE
Narrow peaks

### PR DESCRIPTION
**Pull request by lldelisle**
Hi,

- I corrected a bug for use summit (use summit in the config file and use summits in the code)...
- I added a new test to show how use summit could indeed change the aspect
- I added a vertical bar in the narrowPeaks when plotted as box to be able to see the summits.

### CODE REVIEW

File: temp.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: pygenometracks_test
```
Comment (bgruening) [inline temp.yml]: is that one needed?

Comment (lldelisle) [reply temp.yml]: Oups, hopefully you spotted it.

**Comment by bgruening** [comment]
Do you need a release?

**Comment by lldelisle** [comment]
Well, if you don't mind, I would say yes, then I update the wrapper for galaxy.

**Comment by lldelisle** [comment]
Many thanks

**Comment by lldelisle** [comment]
I do not remember, it will autodeploy or only big releases?

**Comment by bgruening** [comment]
Yes it should, but it was not ... I tried to fix it and found that I missed bumping the version number in the _version.py file. So you be there now. Can you remind an old man next time to bump the version number :)

**Pull request merged**